service/src/main/resources/providers.yml

```diff
@@ -136,6 +143,9 @@ externalcreds:
                                     "https://dev.ukbiobank.terra.bio/#fence-callback",
                                     "https://dev.terra.biodatacatalyst.nhlbi.nih.gov/#fence-callback",
                                     "https://terra.dsde-dev.broadinstitute.org/#fence-callback" ]
+    sage:
+      allowedRedirectUriPatterns: [ "https://bvdp-saturn-dev.appspot.com/ecm-callback" ]
```

Comment:
Would we also need `"http://localhost:3000/ecm-callback"` here?

Reply:
sage did not allow me to add localhost as a redirect

Reply:
Oh interesting!